geopyspark/geotrellis/histogram.py

```diff
@@ -24,6 +26,14 @@ class Histogram(object):
     def __init__(self, scala_histogram):
         self.scala_histogram = scala_histogram
 
+    @classmethod
+    def from_dict(cls, d):
```

Comment:
Sorry to nitpick, but `d` should probably be renamed to something else if this method will be used by users. Also, needs the remaining docstrings.